TicketsDemo.Domain/DefaultImplementations/FileLogger.cs

```diff
@@ -11,19 +11,20 @@ namespace TicketsDemo.Domain.DefaultImplementations
     public class FileLogger : ILogger
     {
         private string _dataFolder;
-
+        private string _logFilePath = "DebugLogs.txt";
```

Comment:
please extract path to file, we log into, to the separate ILoggingConfiguration service